oxd-server/src/main/java/org/gluu/oxd/server/op/GetClientTokenOperation.java

```diff
@@ -31,21 +31,24 @@
 
     private static final Logger LOG = LoggerFactory.getLogger(GetClientTokenOperation.class);
 
+    private OpClientFactory opClientFactory;
     /**
      * Base constructor
      *
      * @param command command
      */
-    protected GetClientTokenOperation(Command command, final Injector injector) {
+    protected GetClientTokenOperation(Command command, final Injector injector, OpClientFactory opClientFactory) {
         super(command, injector, GetClientTokenParams.class);
+        this.opClientFactory = opClientFactory;
```

Comment:
All these references are redundant, add to `BaseOperation`, it's responsibility of guice to construct it.
```
public OpClientFactory getOpClientFactory () {
        return getInstance(OpClientFactory .class);
    }
```